app/src/main/java/org/oppia/app/help/HelpActivity.kt

```diff
@@ -17,20 +17,36 @@ class HelpActivity : InjectableAppCompatActivity(), RouteToFAQListListener {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     activityComponent.inject(this)
-    helpActivityPresenter.handleOnCreate()
+    val isFromExploration = intent.getBooleanExtra(
+      BOOL_IS_FROM_EXPLORATION_EXTRA_KEY,
+      /* defaultValue= */ false
+    )
+    helpActivityPresenter.handleOnCreate(isFromExploration)
     title = getString(R.string.menu_help)
   }
 
   companion object {
-    fun createHelpActivityIntent(context: Context, profileId: Int?): Intent {
+
+    internal const val BOOL_IS_FROM_EXPLORATION_EXTRA_KEY = "BOOL_IS_FROM_EXPLORATION_EXTRA_KEY"
+
+    fun createHelpActivityIntent(
+      context: Context,
+      profileId: Int?,
+      isFromExploration: Boolean
+    ): Intent {
       val intent = Intent(context, HelpActivity::class.java)
       intent.putExtra(KEY_NAVIGATION_PROFILE_ID, profileId)
+      intent.putExtra(BOOL_IS_FROM_EXPLORATION_EXTRA_KEY, isFromExploration)
       return intent
     }
 
     fun getIntentKey(): String {
       return KEY_NAVIGATION_PROFILE_ID
     }
+
+    fun getIsFromExplorationKey(): String {
```

Comment:
Instead  of this function you can directly use `HelpActivity. BOOL_IS_FROM_EXPLORATION_EXTRA_KEY ` whereever needed.

Reply:
Done